Gemfile

```diff
@@ -9,13 +9,14 @@ gem 'rack-cors'
 gem 'rakeup'
 gem 'thin'
 
-gem 'grape', '<0.19.0' # TODO: Grape 1.2.4 reduces performances
+gem 'grape', '>1.4.0', github: 'ruby-grape/grape', branch: 'master'
```

Comment:
Do we need master? What if they push some new commits, each `bundle install` would install different version?